### ISSUE TITLE
Cop idea: `Hash#values_at`

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#12336]
## Description

Use `Hash#values_at` ~~or `Hash#fetch_values`~~ when you need to retrieve several values consecutively from a hash.

https://rubystyle.guide/#hash-values-at-and-hash-fetch-values

## Behavior

```ruby
# bad
email = data['email']
username = data['nickname']

# bad
keys = %w[email nickname].freeze
email, username = keys.map { |key| data[key] }

# good
email, username = data.values_at('email', 'nickname')

# good
keys = %w[email nickname].freeze
email, username = data.values_at(*keys)
```

**Comment by ydakuka** [comment]
I am not sure that this rule does not conflict with the `Style/ParallelAssignment` [cop](https://docs.rubocop.org/rubocop/cops_style.html#styleparallelassignment).

**Comment by koic** [comment]
> I am not sure that this rule does not conflict with the Style/ParallelAssignment [cop](https://docs.rubocop.org/rubocop/cops_style.html#styleparallelassignment).

Good point. It may be beneficial to re-review the https://rubystyle.guide/#hash-values-at rule in the style guide again before suggesting the cop implementation. Actually, it conflicts with the https://rubystyle.guide/#parallel-assignment rule as well.

**Comment by ydakuka** [comment]
Done. https://github.com/rubocop/ruby-style-guide/pull/936
